File: Nível 3/1848 - Corvo Contador/main.py
gritos = 0
soma = 0

while(gritos < 3):
    texto = str(input())

    if(texto == "caw caw"): 
        print(soma)
        soma = 0
        gritos += 1
    else:
        valores = [letra for letra in texto]
        
        if(valores[0] == "*"):
            soma += 4
        if(valores[1] == "*"):
            soma += 2
        if(valores[2] == "*"):
            soma += 1

File: Nível 3/1848 - Corvo Contador/main.lua
gritos = 0
soma = 0

while(gritos < 3) do
    texto = io.read()

    if texto == "caw caw" then
        print(soma)
        soma = 0
        gritos = gritos + 1
    else
        valores = {}

        for i = 1, #texto do
            table.insert(valores, string.sub(texto, i, i))
        end

        if valores[1] == "*" then soma = soma + 4 end
        if valores[2] == "*" then soma = soma + 2 end
        if valores[3] == "*" then soma = soma + 1 end
    end
end